tensorflow/tensorboard/backend/server.py

```diff
@@ -67,21 +68,19 @@ def ParseEventFilesSpec(logdir):
   files = {}
   if logdir is None:
     return files
+  uri_pattern = re.compile("[^:/]+://.*")
```

Comment:
Can we make this:
"[a-zA-Z][0-9a-zA-Z.]*://.*" to match ParseURI?
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/platform/file_system.cc#L82

(Note that function might get moved to core/lib/io/path.cc by the time you see it)